.github/workflows/rack-box-release.yml

```diff
@@ -11,8 +11,29 @@ on:
     types: [ published ]
 
 jobs:
-  setup:
-    runs-on: ubuntu-20.04
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+# docker works on all, but ubuntu is cheapest
+        - builder: docker
+          os: ubuntu-20.04
+          shell: bash
+# hyperv isn't supported yet (https://github.com/actions/virtual-environments/issues/183#issuecomment-706244929)
+#        - builder: hyperv
+#          os: windows-latest
+#          shell: msys2 {0}
+# virtualbox works only on macos (VT-x is not available: VERR_VMK_NO_VMX)
+        - builder: virtualbox
+          os: macos-latest
+          shell: bash
+
+    name: build ${{ matrix.builder }} ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: ${{ matrix.shell }}
```

Comment:
I've simplified the release workflow so that after you get past this part, it is almost identical to the continuous workflow thanks to eliminating the setup job, eliminating some of the caches, and running the same steps in both workflows up to the packer build.